cpp/daal/src/algorithms/kmeans/oneapi/cl_kernels/kmeans_cl_kernels_distr_steps.cl

```diff
@@ -0,0 +1,91 @@
+/* file: kmeans_cl_kernels_distr_steps.cl */
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+/*
+//++
+//  Implementation of K-means OpenCL kernels.
+//--
+*/
+
+#ifndef __KMEANS_CL_KERNELS_DISTR_STEPS_CL__
+#define __KMEANS_CL_KERNELS_DISTR_STEPS_CL__
+
+#include <string.h>
+
+#define DECLARE_SOURCE(name, src) static const char * name = #src;
+
+DECLARE_SOURCE(
+    kmeans_cl_kernels_distr_steps,
+
+    __kernel void init_clusters_distr(__global int * partialCentroidsCounters, __global algorithmFPType * partialCentroids, __global int * cCounters,
```

Comment:
I'm not sure you need to add suffix 'distr' for every function. What the reasons for that?